docs/rest.md

```diff
@@ -10,15 +10,18 @@ subject to change at any time. Feedback through our GitHub issues is appreciated
   - [Bundle Permissions API](#bundle-permissions-api)
   - [Oauth2 API](#oauth2-api)
   - [Users API](#users-api)
-  - [Worksheet Items API](#worksheet-items-api)
+  - [Chatbox API](#chatbox-api)
+  - [Help API](#help-api)
   - [Workers API](#workers-api)
   - [Bundles API](#bundles-api)
+  - [Interpret API](#interpret-api)
```

Comment:
Can we call this `worksheet-interpretation-api` or something (to say it's for worksheets)?